package.json

```diff
@@ -99,7 +99,7 @@
     "lint-staged": "^8.2.0",
     "mocha": "^6.1.4",
     "npm-run-all": "^4.1.5",
-    "prettier": "^1.18.2",
+    "prettier": "~1.18.2",
```

Comment:
Build was failing with the resolved version of `1.19.2` so keeping it at 1.18.x until we upgrade prettier/eslint to latest